src/screens/OtherProfile/index.js

```diff
@@ -1047,39 +897,15 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center'
   },
-  toggleSwitchContainer: {display: 'flex', alignSelf: 'flex-end', paddingVertical: 10},
-  rightHeaderContentContainer: {display: 'flex', flexDirection: 'row'},
-  headerImageContainer: {display: 'flex', flexDirection: 'row', marginBottom: 20},
-  bioAndSendChatContainer: (isAnonimity) => ({
-    backgroundColor: isAnonimity ? colors.bondi_blue : colors.blue1,
-    borderRadius: 15,
-    paddingHorizontal: 10,
-    paddingTop: 10
-  }),
-  bioText: (isAnonimity) => ({
-    color: isAnonimity ? colors.greenDark : colors.white,
-    fontSize: 14,
-    fontWeight: '600',
-    lineHeight: 22
-  })
+  rightHeaderContentContainer: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  headerImageContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 20
+  }
 });
 
-BioAndChat.propTypes = {
-  isAnonimity: PropTypes.bool,
-  bio: PropTypes.string,
-  openBio: PropTypes.bool,
-  isSignedMessageEnabled: PropTypes.bool,
-  showSignedMessageDisableToast: PropTypes.bool,
-  loadingGenerateAnon: PropTypes.bool,
-  avatarUrl: PropTypes.string,
-  anonProfile: PropTypes.object,
-  onSendDM: PropTypes.func,
-  setDMChat: PropTypes.func,
-  loadingSendDM: PropTypes.bool,
-  dmChat: PropTypes.string,
-  username: PropTypes.string,
-  toggleSwitch: PropTypes.func,
-  isAnonimityEnabled: PropTypes.bool
-};
-
 export default withInteractionsManaged(OtherProfile);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [781-909]

The styles `containerBio`, `seeMore`, `bioAndSendChatContainer`, and `bioText` are removed. Also, the `PropTypes` for `BioAndChat` are removed. If these styles are specific to the `BioAndChat` component, they should be defined within the `BioAndChat` component file. Also, as mentioned before, it's a good practice to validate props using `PropTypes`.
```diff
- const styles = StyleSheet.create({
-   // other styles...
-   containerBio: {
-     marginBottom: 10
-   },
-   seeMore: {
-     fontFamily: fonts.inter[500],
-     fontSize: 14,
-     color: colors.black
-   },
-   bioAndSendChatContainer: (isAnonimity) => ({
-     backgroundColor: isAnonimity ? colors.bondi_blue : colors.blue1,
-     borderRadius: 15,
-     paddingHorizontal: 10,
-     paddingTop: 10
-   }),
-   bioText: (isAnonimity) => ({
-     color: isAnonimity ? colors.greenDark : colors.white,
-     fontSize: 14,
-     fontWeight: '600',
-     lineHeight: 22
-   })
- });
- 
- BioAndChat.propTypes = {
-   isAnonimity: PropTypes.bool,
-   bio: PropTypes.string,
-   openBio: PropTypes.bool,
-   isSignedMessageEnabled: PropTypes.bool,
-   showSignedMessageDisableToast: PropTypes.bool,
-   loadingGenerateAnon: PropTypes.bool,
-   avatarUrl: PropTypes.string,
-   anonProfile: PropTypes.object,
-   onSendDM: PropTypes.func,
-   setDMChat: PropTypes.func,
-   loadingSendDM: PropTypes.bool,
-   dmChat: PropTypes.string,
-   username: PropTypes.string,
-   toggleSwitch: PropTypes.func,
-   isAnonimityEnabled: PropTypes.bool
- };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->